types/lesgo/tests/services/S3Service.ts

```diff
@@ -1,14 +1,14 @@
-import S3Service from "lesgo/services/S3Service";
+import S3Service from 'lesgo/services/S3Service';
 
 // $ExpectType S3Service
 const s3 = new S3Service({
-    accessKeyId: "AKID",
-    secretAccessKey: "SECRET",
-    region: "us-west-2",
+	accessKeyId: 'AKID',
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/lesgo/tests/services/S3Service.ts;
  Line: 5;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.